### ISSUE TITLE
[Android] Publish APK Signing Key Hash

### ISSUE THREAD

**Issue by rut8634** [streetwriters/notesnook#7067]
Android users need a way to verify the apk they've downloaded is genuine by verifying the [signing key](https://developer.android.com/studio/publish/app-signing) hash (apk hash is a different matter). GrapheneOS users, especially, are likely accustomed to using [AppVerifier](https://github.com/soupslurpr/AppVerifier) to automate this process, but Notesnook's signing key hash is not present in the internal database, or found elsewhere publicly.

It would be helpful to have the developer's signing key hash posted somewhere on the GH repository, the website, Twitter, or somewhere else relevant.

See an example here, where developers of AppVerifier leave their signing key hash publicly available in their repo, along with a link to the hash on their Bluesky for additional verification and security.

![Image](https://github.com/user-attachments/assets/f6fcfeed-c8b8-450d-88cf-cb3de2b7ad6e)

**Comment by rut8634** [comment]
Any update on this?

**Comment by dtirer** [comment]
I'm also curious about this.  @rut8634 did you ever find it?

**Comment by ryanchuah** [comment]
Would appreciate an update on this as well. This is pretty important for a privacy-focused app

**Comment by astrovolt** [comment]
Any update? Using Obtainium with AppVerifier is a very convenient and useful way to install and update open source apps like Notesnook.
Verifying the signing key adds an **important** safety step pre installation.
Would be amazing to have it listed somewhere in documentation, and/of be added to the AppVerifier database.

**Comment by rut8634** [comment]
> I'm also curious about this. [@rut8634](https://github.com/rut8634) did you ever find it?

Nope, sure didn't. My understanding is that implementing this step would be a pretty quick task, so I'm not sure why it's not being considered. Even an acknowledgement from the developers would be great.

**Comment by Kawys** [comment]
They posted the key hash on their X / Twitter. This is the hash of signing key for GitHub release **only**.

> com.streetwriters.notesnook 23:AF:C1:F8:ED:5F:BB:D0:85:2A:37:22:2D:DC:E0:6F:92:92:71:02:F0:12:6F:99:6A:56:23:F5:27:74:C6:90

[Source](https://x.com/notesnook/status/1994283536682242473)

**Comment by rut8634** [comment]
> They posted the key hash on their X / Twitter. This is the hash of signing key for GitHub release **only**.
> 
> > com.streetwriters.notesnook 23:AF:C1:F8:ED:5F:BB:D0:85:2A:37:22:2D:DC:E0:6F:92:92:71:02:F0:12:6F:99:6A:56:23:F5:27:74:C6:90
> 
> [Source](https://x.com/notesnook/status/1994283536682242473)

Thank you! This is the answer.
